src/services/index.js

```diff
@@ -1,6 +1,5 @@
-export const CONTROLLER_URL = __DEV__
-  ? 'https://dev-logistics-wizard.mybluemix.net/api/v1'
-  : 'https://logistics-wizard.mybluemix.net/api/v1';
+export const CONTROLLER_URL = process.env.CONTROLLER_SERVICE ?
+  `${process.env.CONTROLLER_SERVICE}/api/v1` : 'https://notset/api/v1';
```

Comment:
Can you add this instead to `config/index.js` in the config.globals object, which can then be referenced here.

This also makes it so you don't have to use `transform-inline-environment-variables`

In the webpack config, these globals are injected into the build.

You can make it something like: `__CONTROLLER_API__` and then within `services/index.js` it can be
`export const CONTROLLER_URL = __CONTROLLER_API__ || 'https://dev-logistics-wizard.mybluemix.net/api/v1'`